lib/ansiblelint/formatters/__init__.py

```diff
@@ -143,25 +116,13 @@ def _severity_to_level(severity: str) -> str:
 class ParseableSeverityFormatter(BaseFormatter):
 
     def format(self, match: "MatchError", colored: bool = False) -> str:
```

Comment:
```suggestion
    def format(self, match: "MatchError") -> str:
```

Looks like the colored argument is not used.